ml-agents-envs/setup.py

```diff
@@ -44,6 +44,11 @@ def run(self):
     packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"]),
     zip_safe=False,
     install_requires=[
+        "gym",
+        "cmake",
+        "atari-py",
+        "opencv-python",
```

Comment:
This is a pretty heavyweight package BTW, on some platforms it will need to re-compile opencv.

Reply:
> Outstanding question : Is it okay for us to have a dependency on gym and gym[atari]? One alternative would be to have a try catch in the gym_wrapper at import and raise an error if gym is not installed as well as adding the pip install gym when running our CI tests. Thoughts?

We could make it optional? We could also remove the atari test, this way, no dependency on atari or opencv

Reply:
I agree, you should NOT put these in ml-agents-envs. No preference between gym-unity and another package.